include/lib/coreboot/cbmem_console.h

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
This console functionality is really a driver so would be better in (include)/drivers/coreboot. The other stuff is OK in lib/coreboot though.